src/zenml/utils/source_utils.py

```diff
@@ -393,16 +371,16 @@ def resolve_class(class_: Type[Any], replace_main_module: bool = True) -> str:
         source_path e.g. this.module.Class.
     """
     initial_source = class_.__module__ + "." + class_.__name__
-    if is_standard_source(initial_source):
-        return resolve_standard_source(initial_source)
+    if is_internal_source(initial_source):
+        return initial_source
```

Comment:
So if I get this right, we now no longer add pins anywhere now, right? Why were we doing this in the first place then?